### PR TITLE
turf war design doc

### PR DESCRIPTION
**Pull request by deltanedas**
new sub-gamemode like thieves but for gamers spraying doors to their colours

### CODE REVIEW

**Comment by deltanedas** [comment]
2 things right now:
- should antags be allowed to roll for turf tagger
- should taggers get an objective or not

File: src/en/proposals/deltanedas-turf-war.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Exterminator
```
[the remark this answers is not in the document]
Comment (deltanedas) [reply src/en/proposals/deltanedas-turf-war.md:1]: a

**Comment by mirrorcult** [comment]
i like the idea of tying it to departments. imo the most most most important part of any gangs design is that players need to feel connected to the gang in a real way (people already like to go cargonia atmosia etc so departments is good for that) and they should be nonlethal

> 3 things right now:
> 
> - should antags be allowed to roll for turf tagger
> 

imo no. dont want to mix these and lead to any confusion. this should be strictly a nonantag fun side thing

>
> - should taggers get an objective or not
>

yes, its fun

>
> - should command be allowed to roll since its a bit silly
>

no its a little too silly

**Comment by mirrorcult** [comment]
actually to expound on the last point:
- i think command members should be selectable, just only for their department (rd for sci etc) and not command as a whole
- sec is probably fine, its funny to have the seccies be a gang

**Comment by deltanedas** [comment]
i will probably need to make a department blacklist for it counting station specific anyway :trollface:

**Comment by deltanedas** [comment]
also what about captain :trollface:

**Comment by mirrorcult** [comment]
cap/hop shouldnt be selectable since they dont have another department than command

**Comment by mirrorcult** [comment]
i feel like the selection process should start with which departments to select and only then start selecting ppl within that department

**Comment by deltanedas** [comment]
yes thats how i already have it done, will just have to only add players to the picking lists if they arent cap or hop (without being hardcoded)

**Comment by Tunguso4ka** [comment]
> yes thats how i already have it done, will just have to only add players to the picking lists if they arent cap or hop (without being hardcoded)

HoP is a head of service department. Only cap has no department other then command.

**Comment by HoidC** [comment]
> > yes thats how i already have it done, will just have to only add players to the picking lists if they arent cap or hop (without being hardcoded)
> 
> HoP is a head of service department. Only cap has no department other then command.

they don't really ever do this in actual gameplay

having a uniquely democractic department might be more interesting for this gamemode anyway

**Comment by deltanedas** [comment]
currently there is no service department just civilian and we all know tiders answer to no one

if there was a service department made then hop could tag for service

**Review by mirrorcult — APPROVED** [review]
send it

**Pull request merged**